src/qml/pages/node/PeerDetails.qml

```diff
@@ -0,0 +1,286 @@
+                    key: KeyText { text: qsTr("Address relay"); }
+                    value: Row {
+                        IconButton {
+                            iconLocation: address_relay_row.isAddressRelay ? "image://images/check" : "image://images/cross"
+                            anchors.verticalCenter: parent.verticalCenter
+                        }
+                    }
+                }
+                KeyValueRow {
+                    id: as_row
+                    property string mappedASValue: details.mappedAS
+                    property bool isMappedAS: mappedASValue != "N/A"
+                    key: KeyText {
+                        text: qsTr("Mapped AS");
+                        active: as_row.isMappedAS
+                    }
+
+                    value: Loader {
+                        sourceComponent: as_row.isMappedAS ? mapped_as : not_mapped_as
+                    }
+
+                    Component {
+                        id: mapped_as
+                        ValText { text: as_row.mappedASValue; }
+                    }
+
+                    Component {
+                        id: not_mapped_as
+                        Row {
+                            IconButton {
+                                iconLocation: "image://images/minus";
+                                icon.color: Theme.color.neutral6
+                                anchors.verticalCenter: parent.verticalCenter
+                            }
+                        }
+                    }
+                }
+            }
+
+            CoreText {
+                text: qsTr("Block data");
+                bold: true;
+                font.pixelSize: 18;
+                horizontalAlignment: Qt.AlignLeft;
+                color: Theme.color.neutral9;
+            }
+
+            Column {
+                width: parent.width
+                bottomPadding: 5
+                KeyValueRow { key: KeyText { text: qsTr("Starting block"); } value: ValText { text: details.startingHeight; }}
+                KeyValueRow { key: KeyText { text: qsTr("Synced headers"); } value: ValText { text: details.syncedHeaders; }}
+                KeyValueRow { key: KeyText { text: qsTr("Synced blocks"); } value: ValText { text: details.syncedBlocks; }}
+            }
+
+            CoreText {
+                text: qsTr("Network traffic");
+                bold: true;
+                font.pixelSize: 18;
+                horizontalAlignment: Qt.AlignLeft;
+                color: Theme.color.neutral9;
+            }
+            Column {
+                width: parent.width
+                bottomPadding: 5
+                KeyValueRow {
+                    key: KeyText { text: qsTr("Direction"); }
+                    value: Row {
+                        IconButton {
+                            iconLocation: details.direction === "Inbound" ? "image://images/arrow-down" : "image://images/arrow-up"
+                            icon.height: 9
```

Comment:
```suggestion
                            icon.height: 11
```

Add 2 pixels to compensate for the ~20% padding around the icon source. Makes it closer to the figma and also easier to see.